WooCommerce/WooCommerceUITests/Screens/Login/LoginEmailScreen.swift

```diff
@@ -47,4 +62,35 @@ final class LoginEmailScreen: BaseScreen {
         let emailTextField = XCUIApplication().textFields[ElementStringIDs.emailTextField]
         return emailTextField.value != nil
     }
+
+    func openHelpMenu() -> Bool {
```

Comment:
I actually did think about that! :) I wasn't sure it counted as a screen, but now I know so I will work on that.

Reply:
> because the methods are chained and the Boolean return value can't be chained with that method.

I thought I understood, but it's clear I do not. I cannot seem to fix this no matter what I do. All i have done is moved it down the code :) 

> creating a new screen (page) object for the Help menu,

Done. Thanks for the encouragement!